modules/awesomebook.js

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line import/prefer-default-export
+export class AwesomBooks {
+  bookArr = [];
+
+  input = () => {
+    const name = document.getElementById('name').value;
+    const author = document.getElementById('author').value;
+    this.bookArr.push({ bookName: name, author });
+    localStorage.setItem('books', JSON.stringify(this.bookArr));
+    // eslint-disable-next-line no-restricted-globals
+    location.reload();
```

Comment:
- [ ] This rule can also be enabled. Remove the linter disabling here and change this line:

  ```
  location.reload();
  ```
  to this line:
  ```
  window.location.reload();
  ```